Dockerfile

```diff
@@ -19,11 +19,11 @@ COPY ./db/wait-for /app/wait-for
 
 ADD ./src/main/resources/template /app/template
 
-RUN chmod +x /app/wait-for
-RUN chmod +x /app/verify/verify
+RUN chmod +x /app/wait-for /app/verify/verify  \
```

Comment:
Each **RUN** command is equivalent to executing a new shell, and each **RUN** command creates a layer. Chaining multiple installation steps in one **RUN** command is useful to minimize the size of each layer.